requirements.txt

```diff
@@ -1,4 +1,5 @@
 Twisted==16.6.0
+cryptography==1.9
```

Comment:
Please pin to `2.0.3` instead to benefit from fixes and binary wheels for Linux.

Reply:
Done